experiments/saxpy/openmp/execute_experiment.tpl

```diff
@@ -4,10 +4,30 @@
 #
```

Comment:
Note, with the changes in this PR, there would be no more need for a per-experiment `execute_experiment.tpl`. To reach that, most `experiment/.../ramble.yaml` files and `configs/.../variables.yaml` files need to be tweaked, so these changes are localized to make the behavior easy to experiment with until I do that.